tools/cloud-build/quota-check/bp.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+blueprint_name: quota-check
+
+vars:
+  deployment_name: quota-check
+  project_id:  # Needs to be provided
+  region:      # Needs to be provided
+  zone:        # Needs to be provided
```

Comment:
Nit: change text to "Provided by check.py"

The way it reads now I would expect that I would have to manually edit this blueprint.